plenum/common/moving_average.py

```diff
@@ -0,0 +1,18 @@
+import math
+
+
+class MovingAverage:
```

Comment:
It's better to call it ExponentialMovingAverage

Reply:
Indeed now it's implemented as exponential moving average, but I wanted to leave it generic enough so more complex algorithms could be implemented in future (moving average with momentum, or even something similar to PID-controller or Kalman filter). It's the same reason why I made tests so generic - they capture generic properties instead of checking that step or impulse response is exponential.